community/gitea/APKBUILD

```diff
@@ -30,7 +30,7 @@ prepare() {
 build() {
         cd "$builddir"
 	export GOPATH="$srcdir"
-	export TAGS="tidb sqlite"
+	export TAGS="tidb sqlite sqlite_unlock_notify"
```

Comment:
as you affect build, it needs to bump pkgver

Reply:
Done.